content/kotlin/concepts/datetime/terms/decodeFromHexString/decodeFromHexString.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.decodeFromHexString()'
+Description: 'Exploring the decodeFromHexString method in Kotlin, a tool to handle hex-encoded strings.' 
+Subjects: 
+  - 'Mobile Development'
+  - 'Web Development'
+  - 'Data Science'
+Tags:
+  - 'Kotlin'
```

Comment:
```suggestion
```
We dont have Tags for topics. They need to come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

Reply:
@SSwiniarski - but there is a Kotlin tag in the list. Or we are not using them for terms?

Reply:
We generally haven't used tags for parent topics, as that would be redundant.